code/modules/reqs/supplypacks.dm

```diff
@@ -782,12 +782,12 @@ ARMOR
 /datum/supply_packs/armor/scout_cloak
 	name = "Scout Cloak"
 	contains = list(/obj/item/storage/backpack/marine/satchel/scout_cloak/scout)
-	cost = 50
+	cost = 35
```

Comment:
```suggestion
	cost = 40
```
Scout cloak is stronger than you think, I think 40 points is fair honestly. 50 is too high a bracket for it though, yeah.

Reply:
I know it's strong, I don't underestimate it, it's worth 50 points, its just too expensive to be viable when the scout cloak is best. By the time you can afford 50 points, its too late in the op for you to stealth insurge more most of the time. The 35 cost point allowed it to well, be used sometimes.